db/flyway/db-admin-migrations/V1_2__create_openings_db.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE IF NOT EXISTS OPENINGS (
+    ID UUID PRIMARY KEY,
+    HAS_TECHNICAL_INTERVIEW BOOLEAN NOT NULL,
+    HAS_TECHNICAL_TEST BOOLEAN NOT NULL,
+    PERIOD_COUNT INTEGER NOT NULL,
```

Comment:
You can add constraint : CHECK (PERIOD_COUNT >= 0) to store positive values for this field. Apply this for all INT columns positives.